.github/styles/Google/We.yml

```diff
@@ -0,0 +1,11 @@
+extends: existence
+message: "Try to avoid using first-person plural like '%s'."
+link: 'https://developers.google.com/style/pronouns#personal-pronouns'
+level: warning
+ignorecase: true
+tokens:
+  - we
+  - we'(?:ve|re)
+  - ours?
```

Comment:
does this include "our"?

Reply:
Yeah, the "?" means the "s" is optional.